src/api/component-instance.md

```diff
@@ -251,10 +251,10 @@ Imperative API for creating watchers.
   Stopping the watcher:
 
   ```js
-  const unwatch = this.$watch('a', cb)
+  const stopWatching = this.$watch('a', cb)
```

Comment:
**question**: Could you please provide more context on why this change is necessary?

Reply:
Unwatch didn't pass spellcheck. I didn't realize it was even a word. https://en.m.wiktionary.org/wiki/unwatch it's marked as rare because it isn't used often.

`stopWatching` is much clearer and passes spellcheck.

Reply:
I don't feel strongly about it.